Dockerfile

```diff
@@ -7,29 +7,26 @@ FROM adoptopenjdk/openjdk8:alpine as builder
 
 WORKDIR /app/source
 
-ENV FC_LANG en-US
-ENV LC_CTYPE en_US.UTF-8
+ENV FC_LANG en-US LC_CTYPE en_US.UTF-8
```

Comment:
This should be
```
ENV FC_LANG=en-US LC_CTYPE=en_US.UTF-8
```
Please refer to documentation https://docs.docker.com/engine/reference/builder/#env, especially the note on "Alternative Syntax"
```
The ENV instruction also allows an alternative syntax ENV <key> <value>
...
The alternative syntax is supported for backward compatibility, but discouraged for the reasons outlined above, and may be removed in a future release.
```
I discovered this after reviewing Ansible upgrade log (0.37 --> 0.38)
```
     "expected_env": [
-        "FC_LANG=en-US",
+        "FC_LANG=en-US LC_CTYPE en_US.UTF-8",
         "JAVA_HOME=/opt/java/openjdk",
         "JAVA_TOOL_OPTIONS=-Xmx7g",
-        "JAVA_VERSION=jdk-11.0.7+10",
+        "JAVA_VERSION=jdk-11.0.10+9",
         "LANG=en_US.UTF-8",
         "LANGUAGE=en_US:en",
         "LC_ALL=en_US.UTF-8",
-        "LC_CTYPE=en_US.UTF-8",
```
I'm not sure of the consequences, if any, but it's probably an oversight that should be fixed before it become a problem.

Reply:
Thanks for the head up. This didn't cause any issue as there was a previous layer with these settings in the base image (temurin 11-jre), but your comment is greatly appreciated as it might not be the same in the future
![image](https://user-images.githubusercontent.com/1711649/179543166-92ee7704-5b1e-4466-8b62-934226d45d68.png)

Reply:
PR in place